From b4655b5e7f1e5b87a758dfd7af01a49a40ed148a Mon Sep 17 00:00:00 2001
From: Anthony Pauthonnier <anthony.pauthonnier@sodira-connect.com>
Date: Thu, 13 Feb 2024 17:26:20 +0100

---
 setup.py         | 2 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 422ba54..a74ee0f 100644
--- a/setup.py
+++ b/setup.py
@@ -31,8 +31,8 @@ def find_version(*file_paths):

 install_requires = [
     'cached-property >= 1.2.0, < 2',
-    'docopt >= 0.6.1, < 0.7',
-    'PyYAML >= 3.10, < 4',
+    'docopt >= 0.6.1',
+    'PyYAML >= 3.10',
     'requests >= 2.6.1, != 2.11.0, != 2.12.2, != 2.18.0, < 2.20',
     'texttable >= 0.9.0, < 0.10',
     'websocket-client >= 0.32.0, < 1.0',
-- 
1.9.1

